### DISCUSSION TITLE
New Farmer - PC Rebooted and Can't Restart Plotting

### DISCUSSION THREAD

**Discussion by vascoeagle** [Chia-Network/chia-blockchain#4624] [Q&A]
I was just starting my first plot and I inadvertently shut down my Terminal window.  I'm using Ubuntu Desktop and can not for the life of me how to get back to the main screen to setup my plots.  
I tried the same process as the install
cd chia-blockchain-gui
npm run electron &

but getting App must be installed orin venv

Is there an easy way to get back in?? 

Thanks!

**Comment by ilyakooo0** [comment]
if you are using the GUI I would suggest you download it from the Releases tab on GitHub. It is always up-to-date and does not require the terminal to operate.
